File: app/src/main/java/info/codingalecr/refuel/views/activities/MenuActivity.java
package info.codingalecr.refuel.views.activities;

import info.codingalecr.refuel.R;
import info.codingalecr.refuel.databinding.ActivityMenuBinding;
import info.codingalecr.refuel.helpers.MenuClickHelper;
import info.codingalecr.refuel.views.base.BaseActivity;

public class MenuActivity extends BaseActivity {

    @Override
    public void initObj() {
        getBinding().btnFuelList.setOnClickListener(MenuClickHelper.getRefuelListClickListener(this));
        getBinding().btnSettings.setOnClickListener(MenuClickHelper.getSettingsClickListener(this));
    }

    @Override
    public void initUI() {

    }

    @Override
    public boolean withToolbar() {
        return true;
    }

    @Override
    public int getLayout() {
        return R.layout.activity_menu;
    }

    @Override
    public ActivityMenuBinding getBinding() {
        return (ActivityMenuBinding) getBaseBinding();
    }
}

File: refuel-kotlin/src/main/java/com/codingalecr/refuel_kotlin/views/activities/MenuActivity.kt
package com.codingalecr.refuel_kotlin.views.activities

import android.content.Intent
import com.codingalecr.refuel_kotlin.R
import com.codingalecr.refuel_kotlin.views.base.BaseActivity
import kotlinx.android.synthetic.main.activity_menu.*

class MenuActivity : BaseActivity() {

    override fun withToolbar() = true

    override fun getLayout() = R.layout.activity_menu

    override fun initObj() {
        btn_fuel_list.setOnClickListener {
            val i = Intent(this, FuelListActivity::class.java)
            startActivity(i)
        }
    }

    override fun initViewModel() {
    }

    override fun initObservers() {
    }

    override fun initUI() {
    }
}
